src/components/map/MapComponent.vue

```diff
@@ -50,19 +50,20 @@ import { fixGeometryOfPoi } from '@/services/poi'
 const rennesApp = inject('rennesApp') as RennesApp
 
 const layerStore = useLayersStore()
-const poiStore = usePoiStore()
-const mapStore = useMapStore()
+const poiStore = usePoiParkingStore()
+const map3dStore = useMap3dStore()
+const mapViewPointStore = useMapViewPointStore()
 const lineViewStore = useLineViewsStore()
 const stationsStore = useStationsStore()
 const travelTimesViewStore = useTravelTimesViewStore()
 const viewStore = useViewsStore()
-const componentAboveMapStore = useComponentAboveMapStore()
+const componentAbovemap3dStore = useComponentAboveMapStore()
```

Comment:
Maybe this name can be confusing?
Because the components are not display only above 3D but above 2D map too